File: src/main/java/superstitio/delayHpLose/DelayRemoveDelayHpLoseBlock.java
package superstitio.delayHpLose;

import com.badlogic.gdx.graphics.Color;
import com.evacipated.cardcrawl.mod.stslib.blockmods.AbstractBlockModifier;
import com.megacrit.cardcrawl.cards.AbstractCard;
import com.megacrit.cardcrawl.cards.DamageInfo;
import superstitio.DataManager;
import superstitio.cardModifier.modifiers.AbstractLupaBlock;

public class DelayRemoveDelayHpLoseBlock extends AbstractLupaBlock {

    public static final String ID = DataManager.MakeTextID(DelayRemoveDelayHpLoseBlock.class);

    public DelayRemoveDelayHpLoseBlock() {
        super(ID);
    }

    @Override
    public float onModifyBlock(float block, AbstractCard card) {
        return super.onModifyBlock(block, card);
    }

    @Override
    public AbstractBlockModifier makeCopy() {
        return new DelayRemoveDelayHpLoseBlock();
    }

    @Override
    public Color blockImageColor() {
        return Color.PINK.cpy();
    }
}

File: superstitioMain/java/superstitio/delayHpLose/DelayRemoveDelayHpLoseBlock.kt
package superstitio.delayHpLose

import com.badlogic.gdx.graphics.Color
import com.evacipated.cardcrawl.mod.stslib.blockmods.AbstractBlockModifier
import superstitio.DataManager
import superstitio.cardModifier.modifiers.AbstractLupaBlock

class DelayRemoveDelayHpLoseBlock : AbstractLupaBlock(ID) {
    //    @Override
    //    public float onModifyBlock(float block, AbstractCard card) {
    //        return super.onModifyBlock(block, card);
    //    }
    override fun makeCopy(): AbstractBlockModifier {
        return DelayRemoveDelayHpLoseBlock()
    }

    override fun blockImageColor(): Color {
        return Color.PINK.cpy()
    }

    companion object {
        val ID: String = DataManager.MakeTextID(DelayRemoveDelayHpLoseBlock::class.java)
    }
}